.github/dependabot.yml

```diff
@@ -14,10 +14,8 @@ updates:
     schedule:
       interval: "daily"
     ignore:
-      - dependency-name: "external/apksig"
+      - dependency-name: "external/constexpr-xxh3"
       - dependency-name: "external/debugger-libs"
       - dependency-name: "external/lz4"
-      - dependency-name: "external/nrefactory"
-      - dependency-name: "external/opentk"
       - dependency-name: "external/robin-map"
-      - dependency-name: "external/sqlite"
+      - dependency-name: "external/xxHash"
```

Comment:
@grendello I am assuming we'll want to update the xxhash dependencies manually in the future, and not have dependabot generate PRs for every change in these repos? If not, I will remove these from this update.

Reply:
Oh, I didn't know there was a suppression mechanism like that. Yes, this is exactly what we want! :) Thanks!